src/components/graph-view.js

```diff
@@ -1698,6 +1698,21 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       return;
     }
 
+    const isTouchPad = event.wheelDeltaY
+      ? event.wheelDeltaY !== -3 * event.deltaY
+      : event.deltaMode !== 0;
```

Comment:
is this a standard way to check for touchPad ?

Reply:
it's the only way I found on Chrome. In Firefox, it does not work, I need to look into it more.

Reply:
simplified by always using the same constant